packages/components/text/src/Text.tsx

```diff
@@ -49,6 +50,8 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
   // get the tokens from the theme
   let [tokens, cache] = useTokens(theme);
 
+  const fontMetricsScaleFactors = useFontMetricsScaleFactors();
+
```

Comment:
This calls it for every platform, meaning every platform will get the warning from the other file, right?

Reply:
Honestly, probably better to have a `Platform.OS === 'ios'` guard here too.